tgui/packages/tgui/interfaces/PortraitPicker.js

```diff
@@ -120,10 +122,10 @@ export const PortraitPicker = (props, context) => {
             <Flex.Item mt={1}>
               <NoticeBox info>
                 {multiline`
```

Comment:
Doesn't need the multiline string, you can just shove the text directly into the tag.